bootstrap-joining-demo/build.sbt

```diff
@@ -4,14 +4,29 @@ enablePlugins(JavaServerAppPackaging)
 
 dockerEntrypoint ++= Seq(
   """-Dakka.remote.netty.tcp.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")"""",
-  """-Dakka.cluster.http.management.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")""""
+  """-Dakka.management.http.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")""""
 // THIS WOULD ONLY BE NEEDED IF WE USED THE ASYNC-DNS CODE
 //,
 //  "-Dakka.io.dns.resolver=async-dns",
 //  "-Dakka.io.dns.async-dns.resolve-srv=true",
 //  "-Dakka.io.dns.async-dns.resolv-conf=on"
 )
 
+/*
+libraryDependencies += Seq(
+  // 1) we want to use akka-management
+  "akka-management",
+
+  // 2) we want to use the bootstrap
+  cluster-bootstrap
```

Comment:
not prefixed with `akka-`?

Reply:
yeah, prefixed `akka-management-cluster-bootstrap` even, this was copied from build.sbt so was project name, fixed here now